Dockerfile

```
@@ -20,11 +21,13 @@ RUN cmake CMakeLists.txt && make && cp ./*.a /usr/lib
 
 ADD . /opt/OpenDDS
 
+ARG ACE_CONFIG_OPTION="--doc-group"
+ARG MPC_ROOT="/opt/OpenDDS/ACE_wrappers/MPC"
 RUN cd /opt/OpenDDS && \
-    ./configure --prefix=/usr/local --security --doc-group --std=c++11 && \
+    ./configure --prefix=/usr/local --security --std=c++11 ${ACE_CONFIG_OPTION} && \
     make && \
     make install && \
-    cp -a /opt/OpenDDS/ACE_wrappers/MPC /usr/local/share/ace/MPC
+    cp -a ${MPC_ROOT} /usr/local/share/ace/MPC
```

```suggestion
    cp -a ${MPC_ROOT} /usr/local/share/MPC
```
This shouldn't be in `ace`, but we may want to keep compatibility with old docker image? @jrw972

---

I don't think backward compatibility is an issue.  IMHO, it is highly unlikely that any use of the Docker image actually cares.

My only request is that after this PR is done, go through the "Compile the Messenger Example" of https://opendds.org/quickstart/GettingStartedDocker.html to make sure it still works.

---

I don't know the answer to these, but I'm in favor of `/usr/local/share/MPC`.